.github/workflows/build.yml

```diff
@@ -0,0 +1,219 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  Init:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Create Build Tag
+      run: |
+        export BUILD_TAG=build-$(date -u +'%Y%m%d%H%M')
+        echo "::set-env name=BUILD_TAG::$BUILD_TAG"
+        mkdir -p dist
+        echo -n $BUILD_TAG > dist/tag
+    - name: Upload Artifacts
+      uses: actions/upload-artifact@master
```

Comment:
https://github.com/actions/upload-artifact/issues/41

Reply:
Thanks for the link. Copy-paste from GitHub's own old docs